### ISSUE TITLE
Not working for Ubuntu 20.04

### ISSUE THREAD

**Issue by Chrisx19** [crigroup/openrave-installation#31]
Yo this thing is not working for ubuntu 20.04 man.... WE need update :(

**Comment by Smartfighting1** [comment]
I agree,this thing is not working for ubuntu 20.04. Please update for ubuntu 20.04, thanks!!

**Comment by DizkiBorsuk** [comment]
well, still not working, i tried to change branch to master in installation script and it did not help

**Comment by yurirocha15** [comment]
Can you check if #34 solve this issue?
